udata_croquemort/tests.py

```diff
@@ -90,11 +92,14 @@ def test_returned_metadata(self, httpretty):
             assert res['check:status'] == test_case['status']
             assert res['check:available'] == test_case['available']
             assert isinstance(res['check:date'], datetime)
+            assert res['metadata:content-type'] == 'text/html'
+            assert res['metadata:content-length'] == '2512124'
+            assert res['metadata:charset'] == 'utf-8'
 
     def test_returned_metadata_w_missing_updated(self, httpretty):
         url = self.resource.url
         metadata = {
-            'content-type': 'text/html; charset=utf-8',
+            'content-type': 'text/html',
```

Comment:
Charset is removed from Content-Type on Croquemort https://github.com/etalab/croquemort/pull/202